PythonWebserver/dbSetup.py

```diff
@@ -134,8 +138,62 @@
 );
 """)
 
+# Tournament related tables
+# Note: Added round_number to tournament_matches for easier querying
+cur.execute("""
+CREATE TABLE tournaments (
+    tournament_id SERIAL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    game VARCHAR(50) NOT NULL,
+    rounds INT,
+    status VARCHAR(20) NOT NULL DEFAULT 'pending',
+    created_by INT REFERENCES users(user_id),
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
```

Comment:
Currently the timestamp does not reflect perth timezone, setup configuration has already been updated in compose file through pr #44, can you please change the code when using TIMESTAMP to:
 
```suggestion
    created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
```

Reply:
good point, thanks for suggestion.